### PR TITLE
NOREF: Browser and interface updates

### PR DESCRIPTION
**Pull request by anelsonvirtru**
*  Browser file interface updates with latest SDK
*  Policy fetch updates with latest SDK
* Adding addition of appId
* Some cleanup
* Tested manually, automated tests on their way after

### CODE REVIEW

File: simple/js/dragdrop.js [hunk 1]
```diff
@@ -180,15 +180,21 @@ function handleFileViewer(evt){
...
+
+        const uuid = await client.getPolicyId(decParams);
+        fetchedPolicy = await client.fetchPolicy(uuid);
         fetchedPolicyBuilder = fetchedPolicy.builder();
         togglePolicyElements(true);
         populateUserList(fetchedPolicy.getUsersWithAccess());
-        getById('editingPolicyText').innerHTML = `Editing Policy...`;
+        getById('editingPolicyText').innerHTML = `Editing Policy`;
     }catch(e){
+      console.log(e);
```
Comment (robbtraister) [inline simple/js/dragdrop.js]: might be better to use `console.error` here?

**Review by robbtraister — COMMENTED** [review]
overall, this looks reasonable, but I don't know enough about the project to say either way

File: simple/js/dragdrop.js (hunk 1 above)
Comment (anelsonvirtru) [reply simple/js/dragdrop.js]: Yes, updated this

**Review by jrschumacher — APPROVED** [review]
LGTM

**Pull request merged**